__tests__/utils/fhir.test.ts

```diff
@@ -158,6 +163,85 @@ const TEST_MEASURE_BUNDLE_WITH_EXPANSION: fhir4.Bundle = {
   ]
 };
 
+const RESOURCE_WITH_NO_SUMMARY: fhir4.Resource = {
+  resourceType: 'Prodcedure'
+};
+
+const RESOURCE_WITH_NO_CODE_PATH: fhir4.Resource = {
```

Comment:
I think `Procedure` does have a primary code path. So this example has nothing at the primary code path.

Reply:
I kept this and just changed it to test for a resource without a code and I added a different test that tests for a resource without a primary code path.